### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by Man2-0**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@
 __pycache__
 .venv
 venv
+cicd_start.sh
```
Comment (bandirom) [inline .gitignore:7]: what is the script?

File: alembic/env.py [hunk 2]
```diff
@@ -11,6 +12,7 @@
 config = context.config
 
 config.set_main_option("sqlalchemy.url", settings.database_uri.unicode_string())
+logging.warning(f"{settings.database_uri.unicode_string()=}")
```
Comment (bandirom) [inline alembic/env.py:15]: please remove it :)

File: tests/alembic_test.py
Comment (bandirom) [inline tests/alembic_test.py:1]: I guess we do not need this test at all

File: tests/test_first.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import os
+
+import sqlalchemy
+from dotenv import load_dotenv
+
+from alembic import config, script
+from alembic.runtime import migration
+
+load_dotenv()
```
Comment (bandirom) [inline tests/test_first.py:9]: try to find a way to avoid loading env everytime

File: tests/test_first.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+DATABASE_URI = os.getenv('DATABASE_URI')
```
Comment (bandirom) [inline tests/test_first.py:11]: you have DB in the settings file. Do not need to call env every time

File: pyproject.toml [hunk 5]
```diff
@@ -17,6 +17,7 @@ python-jose = "~=3.3"
 cryptography = "~=42.0"
 passlib = { version="~=1.7", extras = ["bcrypt"] }
 pydantic = { version="~=2.6", extras = ["email"] }
+pytest = "^8.1.1"
```
Comment (bandirom) [inline pyproject.toml:20]: pytest shouldn't be in a main dependencies. Move to the `dev` deps

File: .gitignore (hunk 1 above)
Comment (Man2-0) [reply .gitignore]: it was for pushing changes to the branch to run ci/cd. I removed it now

File: alembic/env.py (hunk 2 above)
Comment (Man2-0) [reply alembic/env.py]: done

File: tests/alembic_test.py
Comment (Man2-0) [reply tests/alembic_test.py:1]: deleted

File: tests/test_first.py (hunk 3 above)
Comment (Man2-0) [reply tests/test_first.py]: i think we don't need env variables there

File: tests/test_first.py (hunk 4 above)
Comment (Man2-0) [reply tests/test_first.py]: removed

File: pyproject.toml (hunk 5 above)
Comment (Man2-0) [reply pyproject.toml]: moved

**Pull request merged**